discover/portal/articles-dxp/140-workflow/01-workflow-designer/00-intro.markdown

```diff
@@ -0,0 +1,52 @@
+# Kaleo Designer [](id=kaleo-designer)
+
+With the proper permissions, users can publish assets. Even if your enterprise
+has the greatest employees in the world, many of the items they'll wish to
+publish must still be reviewed, for a variety of reasons. The Kaleo Designer in
+@product@ lets you design workflow definitions so your assets go through a
```

Comment:
Hey Renato, this article never appears to CE users. Note the above url has "articles-dxp", where it would usually have "articles". The dev.liferay.com publication ignores articles in articles-dxp, but the customer.liferay.com publication process pulls all articles in, the ones in "articles" and the ones in "articles-dxp". So this is okay.Althoughw e are trying to remove the use of this token as much as possible, because it's overused and often redundant. Readers are looking at the documentation for Liferay Portal, so we usually don't need to say it at all. It's assumed. I'll remove this instance of it completely. :+1:

Reply:
Thanks for this clarification, @rbohl! I'll try to take a look on this next time. 😉